app/assets/stylesheets/sufia/_styles.scss

```diff
@@ -75,3 +75,13 @@ label.disabled {
   text-decoration:underline;
   background-color:#0088CC;
 }
+
+.check_all {
+  white-space: nowrap;
+  .caret {
+    margin-left: 0;
+  }
+  input#check_all {
```

Comment:
I'm wondering why we are defining styles for that check box in two different places?  Shouldn't we be doing it all in one block?